components/service/glean/src/test/java/mozilla/components/service/glean/debug/GleanDebugActivityTest.kt

```diff
@@ -88,4 +95,40 @@ class GleanDebugActivityTest {
         assertEquals(testPackageName,
             shadowOf(activity.get()).peekNextStartedActivityForResult().intent.`package`!!)
     }
+
+    @Test
+    fun `pings are sent using sendPing`() {
+        val server = MockWebServer()
+        server.enqueue(MockResponse().setBody("OK"))
+
+        resetGlean(config = Glean.configuration.copy(
+            serverEndpoint = "http://" + server.hostName + ":" + server.port
+        ))
+
+        // Put some metric data in the store, otherwise we won't get a ping out
+        // Define a 'booleanMetric' boolean metric, which will be stored in "store1"
+        val booleanMetric = BooleanMetricType(
+            disabled = false,
+            category = "telemetry",
+            lifetime = Lifetime.Application,
+            name = "boolean_metric",
+            sendInPings = listOf("store1")
+        )
+
+        booleanMetric.set(true)
+
```

Comment:
I'd add an `assertTrue(booleanMetric.testHasValue())` to be safe: that will wait and make sure that there's data there before we attempt to send. Otherwise, depending on CI, we might have intermittents.